crates/connectors/ndc-postgres/src/configuration/version2.rs

```diff
@@ -50,6 +50,17 @@ impl RawConfiguration {
     }
 }
 
+pub fn default_unqualified_schemas() -> Vec<String> {
```

Comment:
I'm not sure I like this approach because it conflates things from different namespaces. Perhaps we need separate settings for unqualified schemas for each of the namespaces?

Reply:
I'm suspecting as much myself.